### ISSUE TITLE
Low temperature reading from PMS5003T

### ISSUE THREAD

**Issue by airgradienthq** [airgradienthq/arduino#178]
There seem to be very low temperature readings sometimes happening from PMS5003T. Model O-1PP

<img width="1305" alt="image" src="https://github.com/airgradienthq/arduino/assets/68736665/349788cd-3033-4e74-9eed-bc1ea7321565">

I think it might make sense to only allow **temperature and humidity readings** within the specs of the sensor:

• Temperature operating range (–40°C to 100°C)
• Relative humidity operating range (0% to 100%)

See: https://www.mouser.com/datasheet/2/1081/ENS210-1984006.pdf

So basically ignoring values outside that band.

**Comment by joshschaeffer** [comment]
Not sure if this helps, but here's how Esphome fixed a similar issue with that sensor:
[https://github.com/esphome/esphome/pull/6100/files](https://github.com/esphome/esphome/pull/6100/files)

**Comment by airgradienthq** [comment]
Will be fixed as part of #190
